<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>黄金矿工</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/images/iii/icon.png</normaloff>:/images/iii/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="lcdNumber_time">
    <property name="geometry">
     <rect>
      <x>563</x>
      <y>15</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>04b</family>
      <pointsize>26</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/iii/title-time.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>60.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newimages/newimage/gold5.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>115</width>
      <height>115</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newimages/newimage/gold5.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newimages/newimage/gold1.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newimages/newimage/gold1.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newimages/newimage/gold1.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newimages/newimage/gold1.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>117</width>
      <height>117</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newimages/newimage/gold5.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newimages/newimage/gold1.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newimages/newimage/gold1.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>596</x>
      <y>146</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newimages/newimage/bluediamond.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_reset">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>15</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/iii/reset.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_break">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>15</y>
      <width>61</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/iii/break.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/iii/miner_init.png);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_begin_background">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>581</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newimages/newimage/beginning.png);</string>
    </property>
    <property name="text">
     <string>begin</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_start">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>230</width>
      <height>95</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/iii/start.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_exit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>420</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/iii/exit.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_rule">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>711</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newimages/newimage/pause.jpeg);</string>
    </property>
    <property name="text">
     <string>pasue</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_finish_background">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>571</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newimages/newimage/over.jpeg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_continue">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>180</width>
      <height>80</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/iii/continue.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>230</y>
      <width>50</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newimages/newimage/pinkpotion.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>360</y>
      <width>50</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newimages/newimage/greenpotion.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>60</y>
      <width>51</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newimages/newimage/time.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_sum">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/iii/title-money.png);</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_restart">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/iii/restart.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_final">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>04b</family>
      <pointsize>79</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>123</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>290</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newimages/newimage/bag.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_14</zorder>
   <zorder>lcdNumber_sum</zorder>
   <zorder>toolButton_reset</zorder>
   <zorder>label_13</zorder>
   <zorder>label_12</zorder>
   <zorder>label_11</zorder>
   <zorder>lcdNumber_time</zorder>
   <zorder>label_1</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>toolButton_break</zorder>
   <zorder>toolButton</zorder>
   <zorder>label_begin_background</zorder>
   <zorder>toolButton_start</zorder>
   <zorder>label_rule</zorder>
   <zorder>toolButton_continue</zorder>
   <zorder>label_finish_background</zorder>
   <zorder>toolButton_exit</zorder>
   <zorder>toolButton_restart</zorder>
   <zorder>label_final</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
